supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -59,7 +59,7 @@
 [discrete]
 [[scheduler]]
 ==== image:images/yes.png[yes] scheduler (noun)
-*Description*: In Red Hat OpenShift, the scheduler is a component of the master that manages the state of the system, places pods on nodes, and ensures that all containers that are expected to be running are actually running.
+*Description*: In Red Hat OpenShift, the _scheduler_ is a component of the control plane that manages the state of the system, places pods on nodes, and ensures that all containers that are expected to be running are actually running.
```

Comment:
> 1. Are we planning to italicise the first occurrence of the term in all glossary entries?
> 

@bredamc Yes we have an effort underway to do this (see #173). I only did it in this PR here because I have a separate PR  that updates this file to add the italics (#238) and it would conflict with this update, so I made the change here instead.

>     2. To avoid ambiguity about whether it is the scheduler or the control plane "that manages ...", perhaps rephrase as "the scheduler is a control-plane component that manages..." (assuming that it is the scheduler that does these tasks!)?

It is indeed the scheduler that performs these tasks. I've reworded as you recommended, but didn't use the hyphen because we don't either in our doc set when we say "control plane component". Thanks!